test/models/retinanet/requirements.txt

```diff
@@ -0,0 +1,89 @@
+absl-py==1.0.0
+antlr4-python3-runtime==4.8
+appdirs==1.4.4
+astunparse==1.6.3
+attrs==21.4.0
+black==22.3.0
+cachetools==5.1.0
+certifi==2022.5.18.1
+charset-normalizer==2.0.12
+click==8.1.3
```

Comment:
Let's remove this file

Reply:
Marked as draft until we resolve this issue